Project 3 Lol

```diff
@@ -0,0 +1,350 @@
+  
+  (c) SARS-CoV-2
+  - object
+  
+  (d) Mexico City
+  - object aggregate made up of objects like inhabitants, buildings, streets, lights, smells, and et cetera.
+  - What I am confused about here is whether or not an object aggregate must be aggregated of all the same object, as this is the case with the examples listed in chapter 5.
+  
+  (e) The trunk of a minivan
+  - fiat object part
+  - page 113 lists "the trunk of your car" as an example of a site.
+  
+  (f) Occupation
+  - Depends, the position, as in the formal title, is a role. 
+  - The conditions and expectations set around a position are generically dependent continuants about the role.
+  - the actual work done, one's occupation of their time, is a process.
+  
+  (g) Ocean
+  - fiat object part
+  
+  (h) Lake
+  - fiat object part
+...
+- My answers for (g) and (h) are informed by the attached article: https://journals.sagepub.com/doi/pdf/10.1177/0263775817700395.
+- I've heard Giacomo and Ali categorize both oceans and lakes as sites according to BFO. I support this categorization.
+  
+```
+7. True or False; explain your answers:
+```
+  (a) An instance of Material Entity can have an instance of Immaterial Entity as part.
+  - True: I think holes and cavities are immaterial entities.
+  - My answer here is informed by chapter 5 of the BFO textbook as well as the attached slides: https://buffalo.app.box.com/s/6wmxnd9vvuhd9ubrldeptaerydqzkbct
+  
+  (b) An instance of Immaterial Entity can have an instance of Material Entity as part.
+  - True: an imaginary friend is in a symmetric parthood relationship with the mind of the child who imagines it.
+  - My take is informed by page 112 of the BFO textbook.
+ ...
+  - Giacomo pointed out how the classes of material and immaterial entities are disjoint, according to BFO.
+  - If this is the case, my answers to both (a) and (b) above are false... am stuck now
+  
+  (c) An organization may have another organization as part.
+  - True: if we assume the opposite ("An organization CANNOT have another organization as part"), then I would never make friends in my classes or work in groups.
+  - Many social organizations only exist insofar as the smaller social organizations constituting them exist as stable entities.
+  
+  (d) An organization may have no members as part. 
+  - True: if nobody attends Anime Club on a given week, it still exists, both as a formal organization and as an empty room.
+  - Unless its founders or relevant organizers dissolve it after weeks of nobody attending, Anime Club continues to exist as an organization with no members.
+  
+  (e) Any site is partially bounded by some instance of Material Entity.
+  - True: a fully immaterial site, one not bounded to anything 'out there', cannot exist in the world. 
+  - I am under the impression that a site is a 'de jure' entity; it only exists because we pay special attention to it.
+  - We cannot demarcate a site around something wholly immaterial. 
+  
```

Comment:
I am not sure about this. The second part of the BFO definition doesn't seem to be really clear on this and may allow for e to be false. But I really think it's up to you and I understand why you would like to have it as true.